### PR TITLE
Fixing unused vars and unchecked errors

### PR DESCRIPTION
**Pull request by 10eMyrT**
This PR comes to fix the Travis CI issue. Two files (`handler.go` and `config.go`) have already been addressed by @Zedjones. The main focus is with `tg.api.Send(newMsg)` in `telegram.go`. I will solve the merge conflicts before the merge.

### CODE REVIEW

File: internal/handlers/telegram/telegram.go [hunk 1]
```diff
@@ -32,7 +32,14 @@ SendMessage sends a message to the Telegram channel specified in the settings
...
 	newMsg := tgbotapi.NewMessage(tg.Settings.ChatID, "")
 	newMsg.Text = msg
-	tg.api.Send(newMsg)
+
+	if _, err := tg.api.Send(newMsg); err != nil {
+		// Keep trying until the message is successfully sent
+		for err != nil {
+			tg.logger.LogError(err)
+			_, err = tg.api.Send(newMsg)
+		}
```
Comment (10eMyrT) [inline internal/handlers/telegram/telegram.go]: This loop forces to keep sending as long as the message did not go through. I thought about setting a number of trials before moving on, and would suggest ***resending 3 times and move on if it still does not go through.***

**Comment by 10eMyrT** [comment]
Conflicts resolved, and Travis checks are successful. We just have to agree on the number of times to try resending.

File: internal/handlers/telegram/telegram.go (hunk 1 above)
Comment (jwflory) [reply internal/handlers/telegram/telegram.go]: @10eMyrT I think three attempts is fair. As long as each failed attempt is logged, which it looks like it is to me.

Comment (10eMyrT) [reply internal/handlers/telegram/telegram.go]: I do too. I have changed the code to consider resending 3 times. Let's wait for other inputs before we merge.

**Comment by 10eMyrT** [comment]
Thanks @jwflory and @Tjzabel for the reviews. I will merge now.

**Pull request merged**